alfasim_sdk/hook_specs.py

```diff
@@ -1333,6 +1333,82 @@ def calculate_entrained_liquid_fraction(
     """
 
 
+def update_internal_deposit_layer(
```

Comment:
For consistency with the nomenclature in ALFAsim, call it `update_internal_deposition_layer`

Reply:
Do it everywhere (function names, variable names and docs)

Reply:
- [ ] change name